util/components.js

```diff
@@ -17,3 +19,86 @@ export async function setOEBInputValueByCSS(driver, cssSelector, value) {
   await oebInputChild.clear()
   await oebInputChild.sendKeys(value);
 }
+
+export async function addNewTag(driver, tagName ){
```

Comment:
I don't think this function makes sense in a file called `components.js`. I don't know what an ideal solution is here, but I have three options in mind:
- Name the function more descriptively, e.g. `badgeAddNewTag`
- Put it into an extra file, e.g. called `badgeHelper.js`
- Leave it in `badge.js` as a separate method

I think the third option makes the most sense, since the idea of `badge.js` was to contain interaction with badge related things. Maybe the file could be split in multiple files, e.g. one `badgeDialog.js`, one `badgeDetails.js`, etc. which then handle the appropriate dialogues of the badge.

Reply:
Thanks for the feedback! I used the second option.